server/utils/jwt.ts

```diff
@@ -96,3 +97,24 @@ export async function getUserForEmailSigninToken(token: string): Promise<User> {
 
   return user;
 }
+
+export async function getAttachmentForJWT(token: string): Promise<Attachment> {
```

Comment:
Now that I see this here it makes me want to move these finders into the relevant model as static methods, eg.

`User.findByEmailSigninToken` and `Attachment.findBySignature`

Reply:
`Attachment.findBySignature` makes sense particularly for the scope of this PR though.